Tests/SourceControlTests/GitRepositoryProviderTests.swift

```diff
@@ -57,23 +57,59 @@ class GitRepositoryProviderTests: XCTestCase {
     }
 
     func testGitShelErrorIsPrintable() throws {
-        let output = "An error from Git"
+        let stdOut = "An error from Git - stdout"
+        let stdErr = "An error from Git - stderr"
         let arguments = ["git", "error"]
         let command = "git error"
         let result = AsyncProcessResult(
             arguments: arguments,
             environment: [:],
             exitStatus: .terminated(code: 1),
-            output: .success(Array(output.utf8)),
-            stderrOutput: .success(Array(output.utf8))
+            output: .success(Array(stdOut.utf8)),
+            stderrOutput: .success(Array(stdErr.utf8))
         )
         let error = GitShellError(result: result)
         let errorString = "\(error)"
         XCTAssertTrue(
-            errorString.contains(output),
-            "Error string '\(errorString)' should contain '\(output)'")
+            errorString.contains(stdOut),
+            "Error string '\(errorString)' should contain '\(stdOut)'")
+        XCTAssertTrue(
+            errorString.contains(stdErr),
+            "Error string '\(errorString)' should contain '\(stdErr)'")
         XCTAssertTrue(
             errorString.contains(command),
             "Error string '\(errorString)' should contain '\(command)'")
         }
+
+    func testGitShelErrorEmptyStdOut() throws {
+        let stdErr = "An error from Git - stderr"
+        let result = AsyncProcessResult(
+            arguments: ["git", "error"],
+            environment: [:],
+            exitStatus: .terminated(code: 1),
+            output: .success([]),
+            stderrOutput: .success(Array(stdErr.utf8))
+        )
+        let error = GitShellError(result: result)
+        let errorString = "\(error)"
+        XCTAssertTrue(
+            errorString.contains(stdErr),
+            "Error string '\(errorString)' should contain '\(stdErr)'")
+        }
+
+    func testGitShelErrorEmptyStdErr() throws {
```

Comment:
typo: `testGitShellErrorEmptyStdErr`

Reply:
Thanks for vigilance. Fixed in [7c2b0ea](https://github.com/swiftlang/swift-package-manager/pull/8152/commits/7c2b0eabaa21d59cf10a34a81914b6f91d4647c3).